omero/developers/Python.rst

```diff
@@ -1241,6 +1253,30 @@ Filesets - added in OMERO 5.0
     fileset = conn.getObject("Fileset", fs_id)
 
 
+Example workflow scripts
```

Comment:
Those workflows create a "dependency" on training-scripts
The ultimate goal is to have our "examples" under https://github.com/ome/omero-guide-python
So better to avoid adding more complexity at that point in time by pulling code from a location that is going to changing.
Better to remove them

Reply:
OK. Removed them.